pkg/app/app.go

```diff
@@ -48,33 +49,53 @@ func App(ctx context.Context, httpClient *http.Client, configItem *config.Items)
 		return err
 	}
 
+	var wg sync.WaitGroup
+	channel := make(chan error, len(configItem.Bins))
+
+	go func() {
+		wg.Wait()
+		close(channel)
+	}()
+
+	wg.Add(len(configItem.Bins))
```

Comment:
Line 64 is the preferred way to do this.  This isn't technically wrong, but 64 is cleaner and easier.